reminders/class_tests/notification_tests.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from datetime import datetime
+from django.utils import timezone
+from django.db import IntegrityError
+from events.models import EventParticipant
+from reminders.models import Notification
+from events.tests import (  # noqa: F401
+    new_event, user0,
+    event_participant_creator as participant0
+)
```

Comment:
Maybe we can use conftest file in the main lets-meet directoy and write there all those fixtures, instead of importing them.
But I think it would be better to make all conftest changes in our project in a different PR, so I opened an issue for that 😄

Reply:
Great suggestion 👍